packages/block-library/src/query/variations.js

```diff
@@ -14,6 +15,27 @@ import {
 } from './icons';
 
 const variations = [
+	{
+		name: '',
+		title: __( 'Posts List' ),
+		icon: postList,
+		attributes: {
+			query: {
+				perPage: 4,
+				pages: 1,
+				offset: 0,
+				postType: 'post',
+				categoryIds: [],
+				tagIds: [],
+				order: 'desc',
+				orderBy: 'date',
+				author: '',
+				search: '',
+				sticky: 'exclude',
```

Comment:
I wanted only to double check whether this change was intended:

```json
 "exclude": [], 
 "sticky": "" 
```

with 

```js
sticky: 'exclude',
```

Reply:
Yes, 'exclude` in sticky is an option of it ('include'|'exclude'|'only').
The `excluded` prop is something I have implemented to auto exclude current entity from showing results.